### ISSUE TITLE
Latest version reported an error, but there were no errors in version 10.4.5

### ISSUE THREAD

**Issue by hjj-lmx** [python-pillow/Pillow#8572]
```python
def pil_to_bytes(pil_img, ext: str,quality: int = 95, exif=None) -> bytes:
    exif = exif or b''
    with io.BytesIO() as output:
        pil_img.save(output, format=ext, exif=exif, quality=quality)
        image_bytes = output.getvalue()
    return image_bytes
```
```pytb
Traceback (most recent call last):
  File "/UD-AI-RemoveObject/ud_cleaner/server_gpu.py", line 235, in process
    image_bytes = pil_to_bytes(
  File "/UD-AI-RemoveObject/ud_cleaner/helper.py", line 144, in pil_to_bytes
    pil_img.save(output, format=ext, exif=exif, quality=quality)
  File "/usr/local/lib/python3.10/dist-packages/PIL/Image.py", line 2605, in save
    save_handler(self, fp, filename)
  File "/usr/local/lib/python3.10/dist-packages/PIL/JpegImagePlugin.py", line 796, in _save
    exif = exif.tobytes()
  File "/usr/local/lib/python3.10/dist-packages/PIL/Image.py", line 4044, in tobytes
    return b"Exif\x00\x00" + head + ifd.tobytes(offset)
  File "/usr/local/lib/python3.10/dist-packages/PIL/TiffImagePlugin.py", line 963, in tobytes
    data = ifd.tobytes(offset)
  File "/usr/local/lib/python3.10/dist-packages/PIL/TiffImagePlugin.py", line 966, in tobytes
    data = self._write_dispatch[typ](self, *values)
  File "/usr/local/lib/python3.10/dist-packages/PIL/TiffImagePlugin.py", line 827, in write_rational
    return b"".join(
  File "/usr/local/lib/python3.10/dist-packages/PIL/TiffImagePlugin.py", line 828, in <genexpr>
    self._pack("2L", *_limit_rational(frac, 2**32 - 1)) for frac in values
  File "/usr/local/lib/python3.10/dist-packages/PIL/TiffImagePlugin.py", line 297, in _limit_rational
    inv = abs(float(val)) > 1
  File "/usr/lib/python3.10/numbers.py", line 291, in __float__
    return int(self.numerator) / int(self.denominator)
ZeroDivisionError: division by zero
```

**Comment by radarhere** [comment]
For clarity, there is no Pillow 10.4.5. I'm guessing you just meant Pillow 10.4.0.

While it would be more helpful to have a copy of the image you are trying to open, this otherwise looks like a duplicate of #8471, which has been resolved by #8474.

The next Pillow release is [scheduled for January 2.](https://github.com/python-pillow/Pillow/blob/main/RELEASING.md)

If you would like to patch your copy of Pillow so that it works in the meantime, you can insert this before your code.
```python
from PIL import TiffImagePlugin

# Patch
def _limit_rational(val, max_val):
    inv = abs(val) > 1
    n_d = TiffImagePlugin.IFDRational(1 / val if inv else val).limit_rational(max_val)
    return n_d[::-1] if inv else n_d
TiffImagePlugin._limit_rational = _limit_rational
```

**Comment by radarhere** [comment]
Pillow 11.1.0 has now been released with the fix for this.
